src/main/java/com/fvv/bookstore/dao/EmployeeDAO.java

```diff
@@ -57,4 +58,26 @@
 	 * @throws DaoException when a problem in database happens.
 	 */
 	Employee findEmployee(final Long id) throws PersonNotFoundException, DaoException;
+	
+	/**
+	 * List the employees by search name.
```

Comment:
look up for javadocs with this "search" word and remove all

Reply:
ok